declarations/Booking.com.json

```
@@ -0,0 +1,11 @@
+{
+  "name": "Booking.com",
+  "documents": {
+    "Terms of Service": {
+      "fetch": "https://admin.booking.com/hotelreg/terms-and-conditions.html?cc1=nl&lang=en",
```

Neither query parameters are strictly necessary, but the `lang` one hopefully enables more stable language selection.
```suggestion
      "fetch": "https://admin.booking.com/hotelreg/terms-and-conditions.html?lang=en",
```